config/locales/de_DE.yml

```diff
@@ -105,7 +105,7 @@ de_DE:
       unauthorized: "Sie haben nicht die notwendigen Rechte, um diese Änderung bei diesem Teilnehmer vorzunehmen."
     recordings:
       title: Serveraufzeichnungen
-      no_recordings: Dieser Server hat keine Aufzeichnungen.
+      no_recordings: Dieser Raum hat keine Aufzeichnungen.
```

Comment:
This change was wrong I think... in this specific context it's actually about recordings on the whole server, not just a specific session or room ("Raum").